File: tasks_001-010/task_5/go/solution_1.go
package solution

func FirstSolution(until int) int {
	for number := until; ; number++ {
		ok := true

		for i := 2; i <= until; i++ {
			if number%i != 0 {
				ok = false
				break
			}
		}

		if ok {
			return number
		}
	}
}

File: tasks_011-020/task_012/js/solution_1.js
const until = 500
let num = 0
let triangle = 0
let divisor = 0

while (true) {
	divisor = 2 // 1 и само число
	triangle += num
	for (let i = 2; i <= Math.sqrt(triangle); i++) {
		if (triangle % i === 0) {
			divisor += 2
			if (Number.isInteger(Math.sqrt(triangle))) divisor--
		}
	}
	if (divisor > until) break
	num++
}